vcu118-run.py

```diff
@@ -371,6 +392,7 @@ def main():
     parser.add_argument("--gdb", default=shutil.which("gdb") or "gdb", help="Path to GDB binary", type=Path)
     parser.add_argument("--openocd", default=shutil.which("openocd") or "openocd", help="Path to openocd binary",
                         type=abspath_arg)
+    parser.add_argument("--n-cores", type=int, default=1, help="Number of harts on bitstream")
```

Comment:
```suggestion
    parser.add_argument("--cores", type=int, default=1, help="Number of harts on bitstream")
```
The -n prefix seems a bit odd, I'd go for `--cores` or `--num-cores`.